libraries/Microsoft.Bot.Configuration/BotConfiguration.cs

```diff
@@ -57,9 +57,9 @@ public BotConfiguration()
         public List<ConnectedService> Services { get; set; } = new List<ConnectedService>();
 
         /// <summary>
-        /// Generate a new key suitable for encrypting
+        /// Generate a new key suitable for encrypting.
         /// </summary>
-        /// <returns>key to use with .Encrypt() method</returns>
+        /// <returns>key to use with .Encrypt() method. </returns>
```

Comment:
Please make this a link to the Encrypt() method, so the docs generate a hyperlink.